### PR TITLE
Fix sct

### PR DESCRIPTION
**Pull request by bgregoir**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -111,6 +111,10 @@
   ([PR #710](https://github.com/jasmin-lang/jasmin/pull/710);
    fixes [#709](https://github.com/jasmin-lang/jasmin/issues/709)).
 
+- Fix bug in speculative constant time checker
+  ([PR #723](https://github.com/jasmin-lang/jasmin/pull/723);
+   fixes [#724](https://github.com/jasmin-lang/jasmin/issues/724)).
+
```
Comment (vbgl) [inline CHANGELOG.md:117]: This entry might be merged with the one announcing the SCT checker.

Comment (bgregoir) [reply CHANGELOG.md:117]: Yes, and in that case, it make no sens to have it.

Comment (vbgl) [reply CHANGELOG.md:117]: You may keep the link to this PR.

Comment (bgregoir) [reply CHANGELOG.md]: I think it is now ready, excepted the horrible history.

**Review by vbgl — APPROVED** [review]

**Pull request merged**